docs/integrations/dbot.md

```diff
@@ -11,19 +11,35 @@ DBot is the Cortex XSOAR machine learning bot which ingests information about in
   "Indicator" : "foo@demi.com",
   "Type": "email",
   "Vendor": "JoeSecurity",
-  "Score": 3
+  "Score": 3,
+  "Reliability": "A: Completely reliable"
 } 
 ```
 
-The DBot score must be at the root level of the context and contain **all** of the keys listed below.
-
-| Key | Meaning |
-| --- | ---|
-| Indicator | Can be: IP, SHA1, MD5, SHA256, Email, or Url |
-| Type | Can be: ip, file, email, or url|
-| Vendor | This is the vendor reporting the score of the indicator|
-| Score | An int representing the status of the indicator. See Score Types below|
-
+The DBot score must be at the root level of the context and contain **all** the required keys as listed below.
+
+| Key | Meaning | Required
+| --- | --- | --- |
+| Indicator | Can be: IP, SHA1, MD5, SHA256, Email, or Url | Required |
+| Type | Can be: ip, file, email, or url | Required |
```

Comment:
@glicht Should I replace the values to match the DBotScoreType?

Reply:
You can